apps-rendering/src/pagination.ts

```diff
@@ -98,20 +98,19 @@ const getNewerPage = (
 	pages: LiveBlock[][],
 	pageNumber: number,
 ): Option<string> => {
-	if (pageNumber > 2) {
+	if (pageNumber > 1) {
 		return maybePageRef(pageNumber - 2, pages);
 	}
 
-	if (pageNumber === 2) {
-		return some('');
-	}
-
 	return none;
 };
 
-const getNewestPage = (pageNumber: number): Option<string> => {
+const getNewestPage = (
+	pages: LiveBlock[][],
+	pageNumber: number,
+): Option<string> => {
 	if (pageNumber > 1) {
-		return some('');
+		return maybePageRef(0, pages);
```

Comment:
Did `some('')` not quite work when testing? Would be curious to know! I think we chose this because, on dcr, when navigating to the newest page we don't need to specify a blockId, which I think results in a more simple request flow.

Reply:
`some('')` was causing issues when testing locally because it wasn't actually deleting the the query parameter `page` from the URL. :(

Reply:
It might be worth looking into this (if there's time!), in this PR we added logic into MAPI for pagination requests: https://github.com/guardian/mobile-apps-api/pull/1785/files. With the query param included in the request we perform some extra logic that isn't necessarily required for the first page. If we could avoid that, I think that'd be good (but not essential). Happy to pair or discuss if I've not been clear :)

Reply:
Interesting, thanks David, we should look into that!

Regarding the `some('')`. Passing an empty string into the pagination component results in an anchor tag with `href=""`. An `href=""` acts as a link to the current page/reloads the current page, rather than deleting the query params and resetting us back to the path-only URL (i.e. the first page).

We suspect the reason this hasn't cropped up on DCR is that it's also appending a `filterKeyEvents` parameter, so it's always setting that string to some variation of `?filterKeyEvents=false` for the first page, rather than using an empty string.

It's likely that AR will have key event filtering at some point (@alinaboghiu ?), so the same thing will apply here. We can decide then whether to remove this solution or keep both params.

**Note:** Having these extra params after navigation back to the first page, for both platforms, likely splits the cache compared to initial hits on the first page (when there are no query params).